docs/build-scripts.md

```diff
@@ -0,0 +1,20 @@
+## Build scripts
+
+### wikiman & pacman
+
+`wikiman` is a MediaWiki specific yaml generator for pacman.
+This needs to be run by developers when updating component versions in `wikiman.yaml`
+
+`pacman` is a generic tool using yaml to fetch a series of codebases and place them on disk.
+This is run as a step in `sync.sh`
+
+### sync.sh
+
+This script will resync the WHOLE git repo.
```

Comment:
This doesn't sync the WHOLE repo; it syncs the `dist` folder

Reply:
```suggestion
This script will resync the `dist` directory.
```